WordPressKit.podspec

```diff
@@ -23,12 +23,11 @@ Pod::Spec.new do |s|
   s.header_dir = 'WordPressKit'
 
   s.dependency 'Alamofire', '~> 4.8.0'
-  s.dependency 'CocoaLumberjack', '~> 3.4'
   s.dependency 'NSObject-SafeExpectations', '0.0.4'
   s.dependency 'wpxmlrpc', '~> 0.9'
   s.dependency 'UIDeviceIdentifier', '~> 2.0'
 
   # Use a loose restriction that allows both production and beta versions, up to the next major version.
   # If you want to update which of these is used, specify it in the host app.
-  s.dependency 'WordPressShared', '~> 1.15-beta' # Don't change this until we hit 2.0 in WPShared
+  s.dependency 'WordPressShared', '~> 2.0.0-beta'
```

Comment:
Have you considered `2.0-beta`? Or is a stricter requirement better while we roll this out?

Reply:
Good catch. This is an oversight, I should've used `2.0-beta`. 👍

Reply:
Addressed here: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/553/commits/308f019fce1247de85e0e122801765494f8b0733